deepchem/models/torch_models/molgan.py

```diff
@@ -0,0 +1,82 @@
+from typing import  List, Tuple #, Any, Optional,
+
+# import torch
+# from torch import nn
+# from torch.nn import functional as F
+# from deepchem.feat.molecule_featurizers.molgan_featurizer import GraphMatrix
+from deepchem.models import WGAN
+# from deepchem.models.layers import MolGANEncoderLayer
+
+
```

Comment:
Can you add unit tests for this new class?

Reply:
I have added the unit test